### DISCUSSION TITLE
Is Windows XDP supported in MsQuic

### DISCUSSION THREAD

**Discussion by zenmiao7** [microsoft/msquic#4230] [General]
Is there any guide doc about deploying MsQuic with Windows XDP? I'm from EXO networking team and I'm curious about the performance improvement brought by XDP.

**Comment by nibanks** [comment]
We don't have any guide yet. We do support Windows usermode MsQuic, but not yet kernel mode. Are you interested in usermode or kernel?

**Comment by zenmiao7** [comment]
user mode is fine.
